federation/v2.0/federation-v2.0.graphql

```diff
@@ -0,0 +1,13 @@
+directive @key(fields: FieldSet!, resolvable: Boolean = true) repeatable on OBJECT | INTERFACE
+directive @requires(fields: FieldSet!) on FIELD_DEFINITION
+directive @provides(fields: FieldSet!) on FIELD_DEFINITION
+directive @external on OBJECT | FIELD_DEFINITION
+directive @shareable on FIELD_DEFINITION | OBJECT
+directive @extends on OBJECT | INTERFACE
+directive @override(from: String!) on FIELD_DEFINITION
+directive @inaccessible on
+  | FIELD_DEFINITION
+  | OBJECT
+  | INTERFACE
+  | UNION
+scalar _FieldSet
```

Comment:
We define the type as `FieldSet` for fed2, without the `_` (there is special code when auto-migrating fed1 schema to fed2 to handle this properly if necessary).

Reply:
done

(we should update it [here too](https://www.apollographql.com/docs/federation/federation-spec/#scalar-_fieldset))